### DISCUSSION TITLE
How to run it in dev to test it in the browser?

### DISCUSSION THREAD

**Discussion by JenuelDev** [cawa-93/vite-electron-builder#357] [Q&A] [answered]
How to run it in dev to test it in the browser?

**Comment by cawa-93** [comment, marked as answer]
`npm run watch` 🤷‍♂️ 

What does you mean by "test it in the browser" ?

**Reply by JenuelDev** [reply]
oh its fine, after searching and understand how electron worls I created my own biolerplate:
https://github.com/BroJenuel-Youtube/vite-vue-electron
